samples/webextension/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "description": "Glean.JS test add-on",
+  "author": "Mozilla",
+  "manifest_version": 2,
+  "name": "gleanjs-test-addon",
+  "version": "0.0.1",
+  "homepage_url": "https://github.com/brizental/gleanjs",
+
+  "permissions": [
+    "storage"
+  ],
+
+  "background": {
+    "scripts": [
+      "glean.js",
```

Comment:
Maybe point to the dist/ folder here instead of copying it all?

Reply:
Nope, it's not possible. Apparently files that we include need to live in a children directory of the one with the manifest.